test/integration/init_int_test.go

```diff
@@ -128,6 +128,21 @@ func (suite *InitIntegrationTestSuite) TestInit_InferLanguageFromUse() {
 	suite.Contains(string(fileutils.ReadFileUnsafe(filepath.Join(ts.Dirs.Work, constants.ConfigFileName))), "language: python3")
 }
 
+func (suite *InitIntegrationTestSuite) TestJSON() {
+	suite.OnlyRunForTags(tagsuite.Init, tagsuite.JSON)
+
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	// Rather than triggering another build/solve on the Platform, just verify the command is
+	// JSON-compatible.
+	cp := ts.Spawn("init", "-o", "json")
+	cp.ExpectLongString(`{"errors":["The following argument is required:`)
+	cp.Expect(`"code":`)
+	cp.ExpectNotExitCode(0)
+	AssertValidJSON(suite.T(), cp)
```

Comment:
This is just verifying that the error reporting is json compatible, rather than the command output itself. Also I don't think it's worth testing for error cases in integration tests unless it's some very specific error case we want to curate.

Reply:
You're right. I will assert proper json error format in a unit test.